From 03553a74b00f52f11beea8816138445106759147 Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
Date: Mon, 2 Jul 2018 18:36:37 -0500
Subject: [PATCH 1753/2302] ASoC: SOF: intel: hda-dai: add include file

Detected with Sparse, variables need to be static if not declared

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
---
 sound/soc/sof/intel/hda-dai.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/sof/intel/hda-dai.c b/sound/soc/sof/intel/hda-dai.c
index 091fcc1..9df9c49 100644
--- a/sound/soc/sof/intel/hda-dai.c
+++ b/sound/soc/sof/intel/hda-dai.c
@@ -10,6 +10,7 @@
 
 #include <sound/pcm_params.h>
 #include "../sof-priv.h"
+#include "hda.h"
 
 #define SKL_FORMATS (SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE | \
 	SNDRV_PCM_FMTBIT_S32_LE)
-- 
2.7.4

